backend/ml/management/commands/ml_train.py

```diff
@@ -50,46 +50,63 @@
 """
 
 
-def fetch_data():
+def fetch_data(trusted=True):
     """Fetches the data from the Comparisons model
 
     Returns:
     - comparison_data: list of
         [   contributor_id: int, video_id_1: int, video_id_2: int,
             criteria: str, score: float, weight: float  ]
     """
-    comparison_data = [
-        [
-            ccs.comparison.user_id,
-            ccs.comparison.entity_1_id,
-            ccs.comparison.entity_2_id,
-            ccs.criteria,
-            ccs.score,
-            ccs.weight,
+    if trusted:
+        comparison_data = [
+            [
+                ccs.comparison.user_id,
+                ccs.comparison.entity_1_id,
+                ccs.comparison.entity_2_id,
+                ccs.criteria,
+                ccs.score,
+                ccs.weight,
+            ]
+            for ccs in ComparisonCriteriaScore.objects
+                .filter(comparison__user__in=User.trusted_users())
+                .prefetch_related("comparison")
+        ]
+    else:
+        comparison_data = [
+            [
+                ccs.comparison.user_id,
+                ccs.comparison.entity_1_id,
+                ccs.comparison.entity_2_id,
+                ccs.criteria,
+                ccs.score,
+                ccs.weight,
+            ]
+            for ccs in ComparisonCriteriaScore.objects
+                .exclude(comparison__user__in=User.trusted_users())
+                .prefetch_related("comparison")
```

Comment:
As discussed during the stand-up: We should run the ML on trusted users, and also on everyone.

Let's avoid the code duplication here:

```suggestion
    comparisons_queryset = ComparisonCriteriaScore.objects.all().prefetch_related("comparison")
    if trusted:
        comparison_queryset = comparison_queryset.filter(comparison__user__in=User.trusted_users())
    comparison_data = [
            [
                ccs.comparison.user_id,
                ccs.comparison.entity_1_id,
                ccs.comparison.entity_2_id,
                ccs.criteria,
                ccs.score,
                ccs.weight,
            ]
            for ccs in comparison_queryset
    ]
```

Maybe this piece above can be done with [django's Queryset.values_list](https://docs.djangoproject.com/en/4.0/ref/models/querysets/#values-list)